sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -51,41 +58,64 @@ export class EventHubConsumerClient {
 
   /**
    * @property
-   * The name of the default consumer group in the Event Hubs service.
+   * The default name for the default consumer group in the Event Hubs service when none is
+   * provided during the creation of the consumer group.
```

Comment:
I'm not sure if this needs to be changed. I couldn't find a way to specify a consumer group on event hubs creation, and it doesn't seem possible to delete the default consumer group (at least via the portal, I get an error from the service).

Reply:
I swear I thought we could change the consumer group name in the portal....
 :(
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/6736/commits/03027f8c2c6138d2b112a489ef2ea785faa54c76.